pulp_container/app/registry_api.py

```diff
@@ -1087,14 +1092,16 @@ def get_content_units_to_add(self, manifest, tag):
             models.MEDIA_TYPE.MANIFEST_LIST,
             models.MEDIA_TYPE.INDEX_OCI,
         ):
-            for listed_manifest in manifest.listed_manifests:
+            for listed_manifest in manifest.listed_manifests.all():
                 add_content_units.append(listed_manifest.pk)
+                add_content_units.append(listed_manifest.config_id)
                 add_content_units.append(listed_manifest.config_blob_id)
```

Comment:
Now, we should not add `config_blob_id` to the list of added content. While appending the content, we should follow the new workflow only. Please, remove the same code a couple of lines below.